tests/contrib/fastapi/test_fastapi.py

```diff
@@ -234,13 +234,14 @@ def test_200_multi_query_string(client, tracer, test_spans):
 
 
 def test_create_item_success(client, tracer, test_spans):
+    _id = "aawlieufghai3w4uhfg"
     response = client.post(
         "/items/",
         headers={"X-Token": "DataDog"},
-        json={"id": "foobar", "name": "Foo Bar", "description": "The Foo Bartenders"},
+        json={"id": _id, "name": "Foo Bar", "description": "The Foo Bartenders"},
```

Comment:
Why do you need to change the ids between these two here? Before was it a bad id because it was the same as the one that had already been created?

Reply:
Ah it's explained in your description, you just want unique ids, got it.